omnisafe/algorithms/model_based/planner.py

```diff
@@ -0,0 +1,916 @@
+# Copyright 2022 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Safe controllers which do a black box optimization incorporating the constraint costs."""
+
+import numpy as np
+import scipy.stats as stats
+import torch
+
+
+class ARCPlanner:  # pylint: disable=too-many-instance-attributes
+    """The Actor Regularized Control (ARC) Planner.
+
+    References:
+        Title: Learning Off-Policy with Online Planning
+
+        Authors: Harshit Sikchi, Wenxuan Zhou, David Held.
+
+        URL: https://arxiv.org/abs/2008.10066
+    """
+
+    # pylint: disable-next=too-many-locals, too-many-arguments
```

Comment:
```suggestion
    # pylint: disable-next=too-many-locals,too-many-arguments
```

No spaces in the pylint comments, same changes for the other parts in this file.